shell/browser/ui/win/jump_list.cc

```diff
@@ -28,6 +28,9 @@ bool AppendTask(const JumpListItem& item, IObjectCollection* collection) {
       FAILED(link->SetDescription(item.description.c_str())))
     return false;
 
+  if (wcslen(item.description.c_str()) > 260)
```

Comment:
Why 260? is this a platform constant?

Also, could we use `.size()` instead of `wcslen`?
```suggestion
  if (item.description.size() > 260)
```

Reply:
260 seems to be a Windows 10 constant. I had to test lengths to find this limit, since I couldn't find it listed anywhere, except for in the [SetDescription](https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-ishelllinka-setdescription) documentation. Here it is mentioned as "`INFOTIPSIZE`", though even if the description exceeds 260 characters, `SetDescription` returns `S_OK`, so a success.

I just ran a test with `size()` and it seems to work as well. Why is this preferred? Should I make a new commit with this change included?

Reply:
@SushiJackal it's preferred because `std::wstring` already has the size of the string stored, so `.size()` is O(1), whereas `wcslen()` is O(n). It's also easier to read :) Please do go ahead and update this PR to use `.size()`.

Reply:
Ooh, alright! Thanks for clearing that up for me. I'll commit right away